src/kernel/chainparams.cpp

```diff
@@ -542,3 +543,33 @@ std::unique_ptr<const CChainParams> CChainParams::TestNet()
 {
     return std::make_unique<const CTestNetParams>();
 }
+
+std::vector<int> CChainParams::GetAvailableSnapshotHeights() const {
```

Comment:
In commit b2cf3f77b14bccf11ad5bc211db7f746ae5073fb
Nit (clang-format-diff): Open braces on new line (here and the other new functions and methods below).

Reply:
done